conditions/src/main/scala/explore/conditions/ConditionsPanel.scala

```diff
@@ -0,0 +1,253 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.conditions
+
+import cats.implicits._
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import react.common._
+import gem.Observation
+import react.semanticui.collections.form.Form
+import react.semanticui.collections.form.FormGroup
+import explore.model.enum.ImageQuality
+import explore.model.enum.CloudCover
+import explore.model.enum.WaterVapor
+import explore.model.enum.SkyBackground
+import react.semanticui.widths._
+import explore.model.Conditions
+import explore.components.forms.EnumSelect
+import explore.components.graphql.SubscriptionRenderMod
+import explore.model.AppStateIO._
+import clue.GraphQLQuery
+import io.circe.{ Decoder, Encoder }
+import io.circe.generic.semiauto.{ deriveDecoder, deriveEncoder }
+import cats.Show
+import gem.util.Enumerated
+import io.circe.HCursor
+import io.circe.DecodingFailure
+import io.circe.JsonObject
+import crystal.react.io.implicits._
+import monocle.macros.Lenses
+import monocle.function.Cons.headOption
+import crystal.react.StreamRendererMod.ModState
+import cats.effect.IO
+import monocle.Lens
+import react.semanticui.elements.button.Button
+import explore.components.undo.Undoer
+
+/*
+query {
+  conditions {
+    observation_id
+  }
+}
+
+mutation {
+  insert_conditions(objects: [{
+    observation_id: "368e5b67-6c1e-4d77-8547-ef16766802fe",
+    cloud_cover: "Any",
+    image_quality: "Any",
+    sky_background: "Any",
+    water_vapor: "Any"
+  }]) {
+    affected_rows
+  }
+}
+ */
+
+final case class ConditionsPanel(
```

Comment:
How's having this in a separate module working? The idea was that we could do development in one part at a time but I don't know how it works in practice

Reply:
Sorry, I haven't tested it much yet. To be honest, I keep forgetting that we have this capability.